packages/fluentui/docs/src/examples/components/Datepicker/index.tsx

```diff
@@ -4,13 +4,15 @@ import Types from './Types';
 import States from './States';
 import Slots from './Slots';
 import Usage from './Usage';
+import Variants from './Variants';
 
 const DatepickerExamples = () => (
   <>
     <Types />
     <States />
     <Slots />
     <Usage />
+    <Variants />
```

Comment:
Please could you change the order of `Usage` and `Variants` blocks?

And align the section name with other examples ;) 
```suggestion
    <Variations />
```

Reply:
Fixed